app/scripts/services/Room.js

```diff
@@ -1,10 +1,13 @@
 (function(){
 	function Room($firebaseArray){
-		var ref = firebase.database().ref().child("rooms");
-		var rooms = $firebaseArray(ref);
+		var ref = new Firebase("https://bloc-chat-ea6f3.firebaseio.com/");
+		var rooms = $firebaseArray(firebaseRef.child('rooms'));
```

Comment:
@SRosenshein this line should be

``` javascript
var rooms = $firebaseArray(ref.child('rooms'));
```
That was my mistake.

2/10: "Too much fire in your code"